### PR TITLE
fix redundant alias and remove unused function

### PR DESCRIPTION
**Pull request by NickrenREN**

### CODE REVIEW

**Comment by NickrenREN** [comment]
@k8s-bot  ok to test

**Comment by vishh** [comment]
@k8s-bot add to whitelist

**Comment by spxtr** [comment]
@vishh there is no whitelist, you have to @k8s-bot ok to test .

**Comment by spxtr** [comment]
@k8s-bot ok to test

**Comment by NickrenREN** [comment]
@vishh  @spxtr   verification build failed
W0106 19:03:03.216] I0106 19:03:03.215727    1709 parse.go:384] type checker error: /usr/local/go/src/net/http/h2_bundle.go:6913:30: undeclared name: hpack
W0106 19:03:03.220] I0106 19:03:03.219672    1709 parse.go:336] type checking encountered some errors in "net/http", but ignoring.
W0106 19:03:03.918] I0106 19:03:03.918221    1709 parse.go:384] type checker error: /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/davecgh/go-spew/spew/bypass.go:33:26: cannot use (*byte)(nil) (value of type *byte) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
W0106 19:03:03.919] I0106 19:03:03.918270    1709 parse.go:384] type checker error: /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/davecgh/go-spew/spew/bypass.go:33:12: unsafe.Sizeof((*byte)(nil)) (value of type uintptr) is not constant
W0106 19:03:03.919] I0106 19:03:03.918889    1709 parse.go:336] type checking encountered some errors in "k8s.io/kubernetes/vendor/github.com/davecgh/go-spew/spew", but ignoring.

I0106 19:09:49.285] Makefile:119: recipe for target 'verify' failed
W0106 19:09:49.385] make: *** [verify] Error 1
W0106 19:09:52.097] Traceback (most recent call last):
W0106 19:09:52.097]   File "/var/lib/jenkins/workspace/pull-kubernetes-verify/./test-infra/jenkins/../scenarios/kubernetes_verify.py", line 101, in <module>
W0106 19:09:52.097]     main(ARGS.branch, ARGS.script, ARGS.force)
W0106 19:09:52.098]   File "/var/lib/jenkins/workspace/pull-kubernetes-verify/./test-infra/jenkins/../scenarios/kubernetes_verify.py", line 85, in main
W0106 19:09:52.098]     'bash', '-c', 'cd kubernetes && %s' % script,
W0106 19:09:52.098]   File "/var/lib/jenkins/workspace/pull-kubernetes-verify/./test-infra/jenkins/../scenarios/kubernetes_verify.py", line 45, in check
W0106 19:09:52.098]     subprocess.check_call(cmd)
W0106 19:09:52.098]   File "/usr/lib/python2.7/subprocess.py", line 540, in check_call
W0106 19:09:52.098]     raise CalledProcessError(retcode, cmd)
W0106 19:09:52.099] subprocess.CalledProcessError: Command '('docker', 'run', '--rm=true', '--privileged=true', '-v', '/var/run/docker.sock:/var/run/docker.sock', '-v', '/etc/localtime:/etc/localtime:ro', '-v', '/var/lib/jenkins/workspace/pull-kubernetes-verify/go/src/k8s.io/kubernetes:/go/src/k8s.io/kubernetes', '-v', '/var/lib/jenkins/workspace/pull-kubernetes-verify/go/src/k8s.io/kubernetes/_artifacts:/workspace/artifacts', '-e', 'KUBE_FORCE_VERIFY_CHECKS=n', '-e', 'KUBE_VERIFY_GIT_BRANCH=master', '-e', 'REPO_DIR=/var/lib/jenkins/workspace/pull-kubernetes-verify/go/src/k8s.io/kubernetes', 'gcr.io/k8s-testimages/kubekins-test:1.6-v20161205-ad918bc', 'bash', '-c', 'cd kubernetes && ./hack/jenkins/verify-dockerized.sh')' returned non-zero exit status 2
E0106 19:09:52.107] Build failed
I0106 19:09:52.107] process 21418 exited with code 1 after 22.3m
E0106 19:09:52.107] FAIL: pull-kubernetes-verify

Maybe not related to the redundant codes I fixed

**Comment by NickrenREN** [comment]
@k8s-bot verify test this

**Comment by vishh** [comment]
@spxtr Oh. I thought a whitelist existed in the past. The current system then is onerous for reviewers.

**Comment by NickrenREN** [comment]
@vishh  remove some redundant alias will cause verification build failed?
W0106 19:03:03.216] I0106 19:03:03.215727 1709 parse.go:384] type checker error: /usr/local/go/src/net/http/h2_bundle.go:6913:30: undeclared name: hpack
W0106 19:03:03.220] I0106 19:03:03.219672 1709 parse.go:336] type checking encountered some errors in "net/http", but ignoring.
W0106 19:03:03.918] I0106 19:03:03.918221 1709 parse.go:384] type checker error: /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/davecgh/go-spew/spew/bypass.go:33:26: cannot use (*byte)(nil) (value of type *byte) as unsafe.ArbitraryType value in argument to unsafe.Sizeof
W0106 19:03:03.919] I0106 19:03:03.918270 1709 parse.go:384] type checker error: /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/vendor/github.com/davecgh/go-spew/spew/bypass.go:33:12: unsafe.Sizeof((*byte)(nil)) (value of type uintptr) is not constant
W0106 19:03:03.919] I0106 19:03:03.918889 1709 parse.go:336] type checking encountered some errors in "k8s.io/kubernetes/vendor/github.com/davecgh/go-spew/spew", but ignoring.

I0106 19:09:49.285] Makefile:119: recipe for target 'verify' failed
W0106 19:09:49.385] make: *** [verify] Error 1
W0106 19:09:52.097] Traceback (most recent call last):
W0106 19:09:52.097] File "/var/lib/jenkins/workspace/pull-kubernetes-verify/./test-infra/jenkins/../scenarios/kubernetes_verify.py", line 101, in 
W0106 19:09:52.097] main(ARGS.branch, ARGS.script, ARGS.force)
W0106 19:09:52.098] File "/var/lib/jenkins/workspace/pull-kubernetes-verify/./test-infra/jenkins/../scenarios/kubernetes_verify.py", line 85, in main
W0106 19:09:52.098] 'bash', '-c', 'cd kubernetes && %s' % script,
W0106 19:09:52.098] File "/var/lib/jenkins/workspace/pull-kubernetes-verify/./test-infra/jenkins/../scenarios/kubernetes_verify.py", line 45, in check
W0106 19:09:52.098] subprocess.check_call(cmd)
W0106 19:09:52.098] File "/usr/lib/python2.7/subprocess.py", line 540, in check_call
W0106 19:09:52.098] raise CalledProcessError(retcode, cmd)
W0106 19:09:52.099] subprocess.CalledProcessError: Command '('docker', 'run', '--rm=true', '--privileged=true', '-v', '/var/run/docker.sock:/var/run/docker.sock', '-v', '/etc/localtime:/etc/localtime:ro', '-v', '/var/lib/jenkins/workspace/pull-kubernetes-verify/go/src/k8s.io/kubernetes:/go/src/k8s.io/kubernetes', '-v', '/var/lib/jenkins/workspace/pull-kubernetes-verify/go/src/k8s.io/kubernetes/_artifacts:/workspace/artifacts', '-e', 'KUBE_FORCE_VERIFY_CHECKS=n', '-e', 'KUBE_VERIFY_GIT_BRANCH=master', '-e', 'REPO_DIR=/var/lib/jenkins/workspace/pull-kubernetes-verify/go/src/k8s.io/kubernetes', 'gcr.io/k8s-testimages/kubekins-test:1.6-v20161205-ad918bc', 'bash', '-c', 'cd kubernetes && ./hack/jenkins/verify-dockerized.sh')' returned non-zero exit status 2
E0106 19:09:52.107] Build failed
I0106 19:09:52.107] process 21418 exited with code 1 after 22.3m
E0106 19:09:52.107] FAIL: pull-kubernetes-verify

Maybe not related to the redundant codes I fixed

Could you help with the review?

**Comment by kargakis** [comment]
@k8s-bot verify test this

File: pkg/controller/garbagecollector/metaonly/types.generated.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ import (
 	pkg3_types "k8s.io/kubernetes/pkg/types"
 	"reflect"
 	"runtime"
-	time "time"
+	"time"
```
Comment (kargakis) [inline pkg/controller/garbagecollector/metaonly/types.generated.go]: This is generated code and I am not sure you should edit it. Mind restoring?

Comment (NickrenREN) [inline pkg/controller/garbagecollector/metaonly/types.generated.go]: Oh, Thanks @kargakis  . I will restore that

**Comment by NickrenREN** [comment]
Review status: 0 of 50 files reviewed at latest revision, 1 unresolved discussion.

---

*[pkg/controller/garbagecollector/metaonly/types.generated.go, line 33 at r1](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/39514#-K_mVAPZPc_YSnQNIWG_-r1-33:-Ka1eulcJ6xKUz9aX-CA:b-896fix) ([raw file](https://github.com/kubernetes/kubernetes/blob/b694702186dba3a0b8b92a7525c78b5ff3995e5c/pkg/controller/garbagecollector/metaonly/types.generated.go#L33)):*
<details><summary><i>Previously, NickrenREN wrote…</i></summary><blockquote>

Oh, Thanks @kargakis  . I will restore that
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.kubernetes.io:443/reviews/kubernetes/kubernetes/39514)*

**Pull request merged**